### DISCUSSION TITLE
V 2.6.1

### DISCUSSION THREAD

**Discussion by Daannoa** [EdgeTX/edgetx#1704] [Q&A] [answered]
Hello, 

I no longer see version 2.6.1 of edgetx in the milestones . 
has it been cancelled?

**Comment by pfeerick** [comment, marked as answer]
Basically, yes, as the release date for 2.7 was 20/03, so there was little point continuing with 2.6.1 given it was delayed... everything 2.6.1 will be in 2.7 anyway, and 2.7 is going to be light on features anyway since it has just been a either a busy or bad time for all involved in one way or another.  2.7 won't release on 20/03, given that is two days away, but will be going into RC soon, so won't be too far away.
